<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 6.1.1 (Python 3.11.2 on win32)" generated="20231127 14:32:38.176" rpa="false" schemaversion="4">
<suite id="s1" name="Tasks" source="C:\Users\apeks\OneDrive\Desktop\my-first-robot\tasks.robot">
<test id="s1-t1" name="Order robots from RobotSpareBin Industries Inc" line="19">
<kw name="Open website to order">
<kw name="Open Available Browser">
<arg>https://robotsparebinindustries.com/#/robot-order</arg>
<msg timestamp="20231127 14:32:38.245" level="FAIL">No keyword with name 'Open Available Browser' found.</msg>
<status status="FAIL" starttime="20231127 14:32:38.245" endtime="20231127 14:32:38.245"/>
</kw>
<kw name="Maximize Browser Window">
<status status="NOT RUN" starttime="20231127 14:32:38.246" endtime="20231127 14:32:38.246"/>
</kw>
<status status="FAIL" starttime="20231127 14:32:38.238" endtime="20231127 14:32:38.246"/>
</kw>
<kw name="Download file">
<status status="NOT RUN" starttime="20231127 14:32:38.250" endtime="20231127 14:32:38.250"/>
</kw>
<kw name="Read csv file as table">
<status status="NOT RUN" starttime="20231127 14:32:38.250" endtime="20231127 14:32:38.250"/>
</kw>
<kw name="Zip the pdfs">
<status status="NOT RUN" starttime="20231127 14:32:38.250" endtime="20231127 14:32:38.250"/>
</kw>
<status status="FAIL" starttime="20231127 14:32:38.238" endtime="20231127 14:32:38.250">No keyword with name 'Open Available Browser' found.</status>
</test>
<doc>Orders robots from RobotSpareBin Industries Inc.
Saves the order HTML receipt as a PDF file.
Saves the screenshot of the ordered robot.
Embeds the screenshot of the robot to the PDF receipt.
Creates ZIP archive of the receipts and the images.</doc>
<status status="FAIL" starttime="20231127 14:32:38.178" endtime="20231127 14:32:38.250"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="Tasks">Tasks</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20231127 14:32:38.195" level="ERROR">Error in file 'C:\Users\apeks\OneDrive\Desktop\my-first-robot\tasks.robot' on line 7: Importing library 'RPA.Browser.Selenium' failed: ModuleNotFoundError: No module named 'RPA'
Traceback (most recent call last):
  None
PYTHONPATH:
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Scripts\robot.exe
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\python311.zip
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\DLLs
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Lib
  C:\Users\apeks\AppData\Local\Programs\Python\Python311
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Lib\site-packages</msg>
<msg timestamp="20231127 14:32:38.206" level="ERROR">Error in file 'C:\Users\apeks\OneDrive\Desktop\my-first-robot\tasks.robot' on line 8: Importing library 'RPA.HTTP' failed: ModuleNotFoundError: No module named 'RPA'
Traceback (most recent call last):
  None
PYTHONPATH:
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Scripts\robot.exe
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\python311.zip
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\DLLs
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Lib
  C:\Users\apeks\AppData\Local\Programs\Python\Python311
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Lib\site-packages</msg>
<msg timestamp="20231127 14:32:38.210" level="ERROR">Error in file 'C:\Users\apeks\OneDrive\Desktop\my-first-robot\tasks.robot' on line 9: Importing library 'RPA.Tables' failed: ModuleNotFoundError: No module named 'RPA'
Traceback (most recent call last):
  None
PYTHONPATH:
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Scripts\robot.exe
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\python311.zip
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\DLLs
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Lib
  C:\Users\apeks\AppData\Local\Programs\Python\Python311
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Lib\site-packages</msg>
<msg timestamp="20231127 14:32:38.225" level="ERROR">Error in file 'C:\Users\apeks\OneDrive\Desktop\my-first-robot\tasks.robot' on line 11: Importing library 'RPA.Desktop' failed: ModuleNotFoundError: No module named 'RPA'
Traceback (most recent call last):
  None
PYTHONPATH:
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Scripts\robot.exe
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\python311.zip
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\DLLs
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Lib
  C:\Users\apeks\AppData\Local\Programs\Python\Python311
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Lib\site-packages</msg>
<msg timestamp="20231127 14:32:38.225" level="ERROR">Error in file 'C:\Users\apeks\OneDrive\Desktop\my-first-robot\tasks.robot' on line 12: Importing library 'RPA.PDF' failed: ModuleNotFoundError: No module named 'RPA'
Traceback (most recent call last):
  None
PYTHONPATH:
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Scripts\robot.exe
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\python311.zip
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\DLLs
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Lib
  C:\Users\apeks\AppData\Local\Programs\Python\Python311
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Lib\site-packages</msg>
<msg timestamp="20231127 14:32:38.230" level="ERROR">Error in file 'C:\Users\apeks\OneDrive\Desktop\my-first-robot\tasks.robot' on line 13: Importing library 'RPA.Smartsheet' failed: ModuleNotFoundError: No module named 'RPA'
Traceback (most recent call last):
  None
PYTHONPATH:
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Scripts\robot.exe
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\python311.zip
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\DLLs
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Lib
  C:\Users\apeks\AppData\Local\Programs\Python\Python311
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Lib\site-packages</msg>
<msg timestamp="20231127 14:32:38.235" level="ERROR">Error in file 'C:\Users\apeks\OneDrive\Desktop\my-first-robot\tasks.robot' on line 14: Importing library 'RPA.Archive' failed: ModuleNotFoundError: No module named 'RPA'
Traceback (most recent call last):
  None
PYTHONPATH:
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Scripts\robot.exe
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\python311.zip
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\DLLs
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Lib
  C:\Users\apeks\AppData\Local\Programs\Python\Python311
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Lib\site-packages</msg>
<msg timestamp="20231127 14:32:38.238" level="ERROR">Error in file 'C:\Users\apeks\OneDrive\Desktop\my-first-robot\tasks.robot' on line 15: Importing library 'RPA.Browser.Selenium' failed: ModuleNotFoundError: No module named 'RPA'
Traceback (most recent call last):
  None
PYTHONPATH:
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Scripts\robot.exe
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\python311.zip
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\DLLs
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Lib
  C:\Users\apeks\AppData\Local\Programs\Python\Python311
  C:\Users\apeks\AppData\Local\Programs\Python\Python311\Lib\site-packages</msg>
</errors>
</robot>
